content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -176,14 +155,47 @@ to manage the lifecycle of Istio.
 
 1. Uninstall steps:
 
+
+## Uninstalling Istio
+
+### Option 1: Uninstall with Helm via `helm template`
```

Comment:
Two headings one after the other are the first indicator that there is a problem with the content's structure. I suggest we split the document into two one document without tiller and another with.
However, I also suggest we do it after 1.1. 
@sdake Do you agree?